src/main/java/io/xpring/ilp/DefaultIlpClient.java

```diff
@@ -0,0 +1,165 @@
+package io.xpring.ilp;
+
+import org.interledger.spsp.server.grpc.AccountServiceGrpc;
+import org.interledger.spsp.server.grpc.BalanceServiceGrpc;
+import org.interledger.spsp.server.grpc.CreateAccountRequest;
+import org.interledger.spsp.server.grpc.CreateAccountResponse;
+import org.interledger.spsp.server.grpc.GetAccountRequest;
+import org.interledger.spsp.server.grpc.GetAccountResponse;
+import org.interledger.spsp.server.grpc.GetBalanceRequest;
+import org.interledger.spsp.server.grpc.GetBalanceResponse;
+import org.interledger.spsp.server.grpc.IlpOverHttpServiceGrpc;
+import org.interledger.spsp.server.grpc.SendPaymentRequest;
+import org.interledger.spsp.server.grpc.SendPaymentResponse;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import io.xpring.ilp.grpc.IlpJwtCallCredentials;
+import io.xpring.xrpl.XpringException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.Optional;
+import java.util.Properties;
+
+/**
+ * A client that can create accounts, get accounts, get balances, and send ILP payments on a connector.
+ */
+public class DefaultIlpClient implements IlpClientDecorator {
+
+    private static String XPRING_ILP_GRPC_URL;
+    private static final Logger logger = LoggerFactory.getLogger(DefaultIlpClient.class);
+
+    // Bring in configuration properties such as GRPC url
+    static {
+        String propertiesFileName = "client.properties";
```

Comment:
Is this standard practice in Java? Are there ways to do environment variable configurations?

Reply:
This is unusual from my perspective, though perhaps it's something that google recommends for bootstrapping grpc?

I would expect to see something like a Factory maybe that can create instances of `DefaultIlpClient` with some sort of properties supplier maybe. It's unclear to me if this is just how XpringSDK works, or if we have some room to make this nicer.

Will discuss verbally with the team today.

Reply:
To align this with `DefaultXpringClient` and per @keefertaylor's comment, I'm going to get rid of this config stuff in favor of a constructor with configuration.